android-core/src/main/java/com/mparticle/internal/UserAudiencesRetriever.kt

```diff
@@ -0,0 +1,20 @@
+package com.mparticle.internal
+
+import com.mparticle.identity.audience.AudienceResponse
+import com.mparticle.identity.audience.AudienceTask
+import com.mparticle.identity.audience.BaseAudienceTask
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+internal class UserAudiencesRetriever(private val apiClient: MParticleApiClient) {
```

Comment:
```suggestion
internal class UserAudiencesRetriever(apiClient: MParticleApiClient) {
```

maybe remove? understand why?